app/src/androidTest/java/org/mozilla/fenix/helpers/Constants.kt

```diff
@@ -18,6 +18,7 @@ object Constants {
     }
 
     const val SPEECH_RECOGNITION = "android.speech.action.RECOGNIZE_SPEECH"
+    const val POCKET_STORIES_UTM_VALUE = "pocket-newtab-android"
```

Comment:
@MozillaNoah Hey Noah, is this likely ever to change? The test below verifies on click of a pocket story item that the URL is that value

Reply:
@Mugurell, I know you were heavily involved in the "Pocket" implementation, could you please share your thoughts ? ☺️

Reply:
I understand the test was failing because it is indeed not guaranteed that the url will contain the publisher's name.
As a workaround to still test some data about the clicked recommended story Andi added a verification of the utm parameter. 
The utm parameter is part of the contract with Pocket and allows them to easily identify and attribute visits - https://github.com/mozilla-mobile/fenix/issues/21599 and so it's important to ensure that is has the right value and is always correctly added by the app.
A small improvement can be made in the proposed change: test the value as an uri parameter: 
- either by converting the string url to an Uri instance and then use something like `url.queryParameter("utm_source").equals("pocket-newtab-android")` 
- either by just concatenating the expected string - `?utm_source=pocket-newtab-android` and then checking if the url contains that.